frontend/TvCv19.Frontend/ClientApp/src/app/phyiscian-admin/phyiscian-admin.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder } from '@angular/forms';
+import { PatientService } from '../patient.service';
+
+@Component({
+  selector: 'app-phyiscian-admin',
+  templateUrl: './phyiscian-admin.component.html',
+  styleUrls: ['./phyiscian-admin.component.scss']
+})
+export class PhyiscianAdminComponent implements OnInit {
+
+  constructor(private formBuilder: FormBuilder, private service: PatientService) { 
+    this.patientRegistrationForm = this.formBuilder.group({
+      name: '',
+      location: '',
+      caregiverId: ''
+    })
+  }
+
+  ngOnInit(): void {
+  }
+
```

Comment:
looks like you forgot to post the form to the server in order to do this you need a onSubmit handler